projects/addon-commerce/components/input-card/test/input-card.component.spec.ts

```diff
@@ -156,6 +156,22 @@ describe(`InputCard`, () => {
     });
 
     describe(`Formatting`, () => {
+        it(`4`, async () => {
+            await testFormat(`1234`, `1234`);
+        });
+
+        it(`7`, async () => {
+            await testFormat(`1234567`, `1234 567`);
+        });
+
+        it(`10`, async () => {
+            await testFormat(`1234567890`, `1234 5678 90`);
+        });
+
+        it(`12`, async () => {
```

Comment:
The card number can have 13, 15, 16, 18 and even 19 digits. The most common option is 16 digits.
Do we have test suites for that?

Reply:
Yep, few lines below this comment)